jetty-ee8/jetty-ee8-servlet/pom.xml

```diff
@@ -74,6 +74,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
+          <!-- Reuse Forks causes test failures -->
+          <reuseForks>false</reuseForks>
```

Comment:
What are the tests that are failing without this? You should probably open some issue about them.

Reply:
Done, opened issue #11957